admin/class-webchangedetector-autoupdates.php

```
@@ -267,39 +554,61 @@ public function wp_maybe_auto_update() {
 			! array_key_exists( 'auto_update_checks_' . $todays_weekday, $auto_update_settings ) ||
 			empty( $auto_update_settings[ 'auto_update_checks_' . $todays_weekday ] )
 		) {
-			WebChangeDetector_Admin::error_log( 'Canceling auto updates: ' . $todays_weekday . ' is disabled.' );
+			\WebChangeDetector\WebChangeDetector_Admin_Utils::log_error( 'Canceling auto updates: ' . $todays_weekday . ' is disabled.', 'wp_maybe_auto_update', 'debug' );
 			$this->set_lock();
 			return;
 		}
 
 		// Get the current time in the same format (HH:MM).
 		$current_time = current_time( 'H:i' );
```

PHPMD (2.15.0)

Avoid unused local variables such as '$current_time'. (Unused Code Rules)

(UnusedLocalVariable)